docs/pages/management/admin/troubleshooting.mdx

```diff
@@ -18,6 +18,29 @@ services such as the Application Service and Database Service.
 
 ## Step 1/3. Enable verbose logging
 
+<Tabs>
```

Comment:
In this Tabs component, I don't think there's a way for the user to tell which tab to choose without looking at both tabs. I would either:

- Explain when a user would choose a particular tab before the Tabs component
- Include both possibilities as separate H3 sections and provide context before the first H3.

Reply:
I've added a short description of both methods.